ironfish/src/network/peers/peerManager.ts

```diff
@@ -1478,6 +1490,8 @@ export class PeerManager {
 
     let changed = false
 
+    this.addressManager.addAddressesFromPeerList(peerList)
```

Comment:
We have a filter a couple lines down to make sure we filter ourselves out of the new peer set, and it'd probably be worth filtering ourselves out of the address manager as well. Would it make sense to name this something more generic, like `addPossiblePeerAddresses()`, and have the function take basically whatever the type of `newPeerSet.values()` is? Then this could be moved down below the `newPeerSet.delete` call.

Reply:
yeah we could do more to also just use the filtering that happens below but I don't think we need to do that.

Reply:
I'm removing the `possiblePeers` store as I'm not so sure of its utility anymore; `addAddressesFromPeerList()` was only used to add to the possible peer store, so I'll remove this.